src/assets/contributors-map.geojson

```diff
@@ -191,8 +191,8 @@
       "geometry": {
         "type": "Point",
         "coordinates": [
-          49.90644802932367, 
-          -123.16229248871166
+          -123.16229248871166,
+          49.90644802932367
```

Comment:
@cammcinnes your pull request picked up other changes for some reason https://github.com/OpenBeta/open-tacos/pull/1144/files
The simplest way to fix this is abandon this PR and start a new one :)